### PR TITLE
[WALLPAPER] Rename Soft Blue to Mizu

### PR DESCRIPTION
**Pull request by foxlet**
## Purpose

A cleanup of the filenames for Mizu's wallpapers (the designs were made prior to its theme).

## Proposed changes

- Rename existing `ReactOS_Soft_Blue` wallpapers to `ReactOS_Mizu`
- Add the press/screenshot version of the wallpaper

### CODE REVIEW

File: modules/wallpapers/ReadMe.txt [hunk 1]
```diff
@@ -28,4 +29,4 @@ To include the module in your build folder, run the configure script with the fl
 
 # For UNIX users
 
-    ./configure.sh -DENABLE_WALLPAPERS=1
\ No newline at end of file
+    ./configure.sh -DENABLE_WALLPAPERS=1
```
Comment (HBelusca) [inline modules/wallpapers/ReadMe.txt]: What has been changed there?

Comment (foxlet) [reply modules/wallpapers/ReadMe.txt]: That line shouldn't be different, it might be a git bug.

Comment (foxlet) [reply modules/wallpapers/ReadMe.txt]: Ah, the original readme has no newline at the end, my editor appended it.

**Comment by Extravert-ir** [comment]
What's the idea behind having 16:9 A and B wallpapers?

**Comment by foxlet** [comment]
To avoid the clashing with the version string on some screen sizes.

**Pull request merged**